staging/kos/pkg/registry/network/subnet/strategy.go

```diff
@@ -167,15 +137,13 @@ func (ss *subnetStrategy) checkNSAndCIDRConflicts(candidate *subnet.Summary, mal
 		return
 	}
 	glog.V(5).Infof("Found %d subnets with vni %d", len(potentialRivals), candidate.VNI)
+	// Check whether there are Namespace and CIDR conflicts with other subnets.
 	for _, potentialRival := range potentialRivals {
-		pr, err := subnet.NewSummary(potentialRival)
-		if err != nil || candidate.SameSubnetAs(pr) {
-			if err != nil {
-				glog.Errorf("failed to parse subnet %s with vni %d: %s", pr.NamespacedName, candidate.VNI, err.Error())
-			}
-			continue
-		}
-		glog.V(6).Infof("Validating %s against %s", candidate.NamespacedName, pr.NamespacedName)
+		// Ignore the error because a malformed subnet would not have been
+		// allowed: it's the code in this file that performs validation.
+		pr, _ := subnet.NewSummary(potentialRival)
```

Comment:
A PR with additional tweaks will follow: the error will be checked. Ignoring it does not account for cases where this version of the API server is rolled out after a more lenient version which lets malformed Subnets through.

Reply:
I would not worry about that, we are not in production.